demo/app.component.ts

```diff
@@ -107,6 +108,25 @@ export class AppComponent implements OnInit {
     this.form2 = this.formBuilder.group({ 'range': [ [ 2, 8 ] ] });
   }
 
+  public onSliderInit(slider: any) {
+    this.sliderObject  = slider;
+    console.debug('Slider Object Instance Initiated and available for use in your component');
+    console.debug(this.sliderObject);
+  }
+
+  public resetSlider() {
+    this.sliderObject.reset();
+  }
+
+  public togglePips() {
```

Comment:
Turns out the `nativeElement` is not necessary because this works:
```javascript
this.mySlider.slider.updateOptions({range: {min: 0, max: newMax}});
```
THANKS!!!!